releng/corundum-src/macros/build-configuration.xml

```diff
@@ -0,0 +1,31 @@
+<!-- 
+ ******************************************************************************
+ * Copyright (c) 2016 Oracle
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0

```

Comment:
Presumably it's fine for us to be copy-pasting these?

Reply:
It's copy/paste with adhering to the license, which is also EPL.